ctype/time.go

```diff
@@ -10,6 +11,10 @@ type Time struct {
 }
 
 func (t *Time) Time() *time.Time {
+	if t.validatedTime == nil {
```

Comment:
don't need this code. 
already exists in Unmarshal
`t.validatedTime = &time.Time{}`